src/commands/message/sign_nep413/mod.rs

```diff
@@ -0,0 +1,152 @@
+use near_crypto::{SecretKey, Signature};
+use near_primitives::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_primitives::hash::hash;
+use serde::Serialize;
+use strum::{EnumDiscriminants, EnumIter, EnumMessage};
+
+pub mod signature_options;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(input_context = crate::GlobalContext)]
+#[interactive_clap(output_context = SignNep413Context)]
+pub struct SignNep413 {
+    #[interactive_clap(value_enum)]
+    #[interactive_clap(skip_default_input_arg)]
+    /// Message encoding:
+    encoding: MessageEncoding,
+    /// The message to sign:
+    message: String,
```

Comment:
Translate it into an intermediate enum like here: https://github.com/Gmin2/near-cli-rs/blob/3200c937eec81d61e24a2a53556a850039d55cde/src/commands/contract/view_storage/keys_to_view/mod.rs